check/pytest

```diff
@@ -19,7 +19,7 @@ cd "$(git rev-parse --show-toplevel)"
 
 PYTEST_ARGS=()
 ACTUALLY_QUIET=""
-for arg in $@; do
+for arg in "$@"; do
```

Comment:
why is this changed? are you sure?

Reply:
$@ splits quoted args on whitespace so the option `-m "not slow"` gets converted to '-m' 'not' 'slow', rather than '-m' 'not slow'. At least I couldn't figure out how to pass in quoted args otherwise. Happy to be schooled on bash though